src/dos/dos_keyboard_layout.cpp

```diff
@@ -929,7 +929,7 @@ KeyboardLayoutResult DOS_LoadKeyboardLayout(const std::string& keyboard_layout,
 		assert(!new_keyboard_layout.empty());
 		LOG_MSG("LOCALE: Loaded keyboard layout '%s' - '%s'",
 		        new_keyboard_layout.c_str(),
-		        DOS_GetKeyboardLayoutName(new_keyboard_layout).c_str());
+		        DOS_GetKeyboardLayoutNameForLog(new_keyboard_layout).c_str());
```

Comment:
`DOS_GetEnglishKeyboardLayoutName` would be a lot better; "for log" doesn't describe what the function is doing. Then you can put an extra comment there saying currently this is only used for logging purposes.

Reply:
Changed. Also renamed one other function to keep the code consistent.